bioptim/dynamics/configure_problem.py

```diff
@@ -671,6 +690,12 @@ def define_cx(n_col: int) -> list:
                     combine_to=f"{name}_states" if as_states and combine_state_control_plot else combine_name,
                 )
 
+        if as_states_dot:
+            n_cx = nlp.ode_solver.polynomial_degree + 1 if isinstance(nlp.ode_solver, OdeSolver.COLLOCATION) else 2
+            cx = define_cx(n_col=n_cx)
+
+            nlp.states_dot.append(name, cx, mx_states_dot, nlp.variable_mappings[name])
+
     @staticmethod
     def configure_q(nlp, as_states: bool, as_controls: bool):
```

Comment:
Similarly to `as_controls`. I would add `as_states_dot` even though it currently does not make much sense in order to standardize the method signatures

Reply:
done.